_data/blocks/big-data-and-data-engineering.pt_BR.yaml

```
@@ -67,3 +69,28 @@ alura-contents:
   - type: COURSE
     title: "Curso Spark: streaming de dados"
     link: https://cursos.alura.com.br/course/spark-streaming-dados
+  - type: COURSE
+    title: "Curso Spark: streaming de dados"
+    link: https://cursos.alura.com.br/course/spark-streaming-dados
+  - type: COURSE
+    title: "Formação inicial de engenharia de dado"
+    link: https://cursos.alura.com.br/formacao-iniciando-engenharia-dados
+movies:
+  - type: MOVIE
+    title: "O Homem Que Mudou o Jogo"
+    link: https://www.netflix.com/br/title/70201437
+  - type: FILME
+    title: "A era dos dados"
+    link: https://www.netflix.com/br/title/81031737
+  - type: FILME
+    title: "O Dilema das Redes"
+    link: https://www.netflix.com/br-en/title/81254224
+  - type: FILME
+    title: "Eu, Robô"
+    link: https://www.telecineplay.com.br/filme/Eu_Robo_6419
+  - type: FILME
+    title: "O Jogo da Imitação"
+    link: https://www.primevideo.com/detail/O-jogo-da-imita%C3%A7%C3%A3o/0RE594TD30MV9KITHPFBQ312IA?_encoding=UTF8&language=pt_BR
```

```suggestion
  - type: MOVIE
    title: "A era dos dados"
    link: https://www.netflix.com/br/title/81031737
  - type: MOVIE
    title: "O Dilema das Redes"
    link: https://www.netflix.com/br-en/title/81254224
  - type: MOVIE
    title: "Eu, Robô"
    link: https://www.telecineplay.com.br/filme/Eu_Robo_6419
  - type: MOVIE
    title: "O Jogo da Imitação"
    link: https://www.primevideo.com/detail/O-jogo-da-imita%C3%A7%C3%A3o/0RE594TD30MV9KITHPFBQ312IA?_encoding=UTF8&language=pt_BR
```

Acredito que esses aqui tambem precisam ser definidos como `MOVIE`, certo?